batchtable.py

```diff
@@ -55,15 +55,15 @@ def loadJSONBatch(self, data_in, object_wise = True):
 		else:
 			self.batch_in = data_in
 		if len(self.batch_in):
-			first_key = self.batch_in.keys()[0]
+			first_key = list(self.batch_in.keys())[0]
```

Comment:
Is there a more efficient way to get the first key without constructing the whole list in memory?

Reply:
Yes.
```suggestion
			first_key = next(iter(self.batch_in))
```

Reply:
Done in a2c2e00